src/content/self-hosted/install/releases.mdx

```diff
@@ -6,18 +6,57 @@ id: releases
 ---
 
 ## 1.8.0
-TBD
-
-### Features
-- Add support for Kubernetes [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md)
-- Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto.
+10 May, 2023
 
 ### Breaking changes
+
 - The default value for the buildkit service session affinity is now `None`. To keep using `ClientIP` as `sessionAffinity` you need to set this value in your [buildkit configuration](self-hosted/administration/configuration.mdx#buildkit).
 
 ### Deprecation Notice
+
 - Support for Kubernetes [1.22](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md) has been deprecated and will be removed next release.
```

Comment:
```suggestion
- Support for Kubernetes [1.22](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md) has been deprecated and will be removed next release (1.9).
```

Assuming 1.9 is the correct version we're referencing here, I think it's helpful to explicitly call this out. Non-blocking suggestion though